deps/v8/src/flag-definitions.h

```diff
@@ -465,7 +465,7 @@ DEFINE_BOOL(turbo_loop_peeling, true, "Turbofan loop peeling")
 DEFINE_BOOL(turbo_loop_variable, true, "Turbofan loop variable optimization")
 DEFINE_BOOL(turbo_cf_optimization, true, "optimize control flow in TurboFan")
 DEFINE_BOOL(turbo_frame_elision, true, "elide frames in TurboFan")
-DEFINE_BOOL(turbo_escape, true, "enable escape analysis")
+DEFINE_BOOL(turbo_escape, false, "enable escape analysis")
```

Comment:
This is https://bugs.chromium.org/p/chromium/issues/detail?id=765433, currently under embargo.

Doesn't affect Node.js in that it depends on executing untrusted code (which no one should do.)

Reply:
Do you think we should revert this change?

Reply:
Not at all, I only commented to add some context.